README.md

```diff
@@ -13,12 +13,17 @@
 
 `avail-light` is a data availability light client that can do the following:
 
-* Listen for newly produced blocks.
+* Listen for newly mined blocks
 * Gain confidence for `N` *cells*, where *cell* is defined as a `{row, col}` pair.
-  > As soon as a new block is available, the light client attempts to gain confidence by asking for a proof from a full 
-  client via JSON-RPC.
-* Batch processces blocks in reverse order (i.e. prioritizes the latest blocks) for lower numbered blocks where no confidence 
-  is yet gained.
+  > As soon as a new block is available, the light client attempts to gain confidence by asking for a proof from a full client via JSON-RPC.
+* Batch processces blocks in reverse order (i.e. prioritizes the latest blocks) for lower numbered blocks 	where no confidence is yet gained.
```

Comment:
This is currently happening on v1?

Reply:
oh rebasing did a mistake 😓  will do the correction